decidim-system/app/views/decidim/system/organizations/_smtp_settings.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="fieldset">
+  <h4><%= t("decidim.system.models.organization.fields.smtp_settings") %></h4>
+  <%= f.fields_for :smtp_settings do |fields| %>
```

Comment:
Shouldn't we add the other [options from mailers](https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration)?
At least:
* authentication
* enable_starttls_auto
* openssl_verify_mode (adding a security warning if "OpenSSL::SSL::VERIFY_NONE")

Reply:
We are using the default settings of actionmailer, in this case those setting are:

authentication: plain (default)
enable_starttls_auto: true (default)
openssl_verify_mode: only accept valid SSL certificates (signed by a CA authority)